feature-extensions/apigee/setup.py

```diff
@@ -263,42 +263,42 @@ def getAWSAccountID():
           colors.ENDC)
     return subprocess.check_output([
         'aws', 'sts', 'get-caller-identity', '--output', 'text', '--query', 'Account'
-    ]).rstrip()
+    ], encoding='UTF-8').rstrip()
 
 
 def collect_userinputs(args):
     if not args.scm_repo:
-        args.scm_repo = raw_input("Please enter the SCM Repo: ").strip()
+        args.scm_repo = input("Please enter the SCM Repo: ").strip()
```

Comment:
So if we are rewriting this to properly match the mainline installer, can we also use the `click` CLI library to collect user input, like the mainline installer does, so this feature pack can be run both interactively by a person AND via a script?

All feature packs going forward should use `click` in the same manner as the mainline installer, so if we are attempting to redo this feature pack the right way we need to do it here as well.